js/client-ladder.js

```diff
@@ -104,6 +104,7 @@
 			}, this);
```

Comment:
You don't seem to have added the events table entry yet.

Reply:
i couldnt commit it through the Web UI... I'm doing it locally and testing